cfme/tests/infrastructure/test_host.py

```diff
@@ -526,26 +530,100 @@ def test_add_ipmi_refresh(appliance, setup_provider):
 
 
 @test_requirements.infra_hosts
-@pytest.mark.parametrize("crud_action", ['edit_from_hosts', 'edit_from_details', 'remove'])
-def test_infrastructure_hosts_crud(appliance, setup_provider, crud_action):
+@pytest.mark.meta(automates=[1634794])
+def test_infrastructure_hosts_crud(appliance, setup_provider):
     """
     Polarion:
         assignee: prichard
         casecomponent: Infra
         caseimportance: low
         initialEstimate: 1/6h
+    Bugzilla:
+        1634794
     """
     host = appliance.collections.hosts.all()[0]
-    if crud_action != 'remove':
-        stamp = fauxfactory.gen_alphanumeric()
-        new_custom_id = f'Edit host data. {stamp}'
 
-        with update(host, from_details=(crud_action == 'edit_from_details')):
-            host.custom_ident = new_custom_id
+    # Case1 - edit from Hosts
+    new_custom_id = f'Edit host data. {fauxfactory.gen_alphanumeric()}'
+    with update(host,
+                from_details=False,
+                cancel=False,
```

Comment:
This test flow looks a lot better IMO! Thanks for making the switch.

Consider not passing cancel kwarg here, as `False` is the default. Optional change, being explicit is good too.

Reply:
Yeah. I knew it wasn't required, but I thought it made the test clearer. I could go either way.

Reply:
I am going to remove the instances of cancel=False. I believe the logic can be understood without them. Thanks.

Reply:
I went ahead and removed the cancel=False also from host.delete. Especially with the comment in the test, the host.delete logic should be clear.